src/Context/LikedVideosContext.jsx

```diff
@@ -0,0 +1,68 @@
+import { createContext, useContext, useReducer, useState } from "react";
+import axios from "axios";
+import { useAlert } from "./AlertContext";
+import { v4 as uuid } from "uuid";
+import { actionAddToLikedVideosList } from "../Helpers/Services/actions";
+
+
+//CONTEXT CREATED
+const LikedVideosContext = createContext();
+
+const LikedVideosProvider = ({ children }) => {
+
+    const encodedToken = localStorage.getItem("authData");
+    const [likedVideos , setLikedVideos] = useState([]);
+    const {alertContent , setAlertContent} = useAlert();
+
+    const addToLikedVideos = async (video) => {
+        try {
+          const response = await axios.post(
+            "/user/likes",
+            { video },
+            {
+              headers: {
+                authorization: encodedToken,
+              },
+            }
+          );
+          if (response.status === 201) {
+            setLikedVideos(response.data.likes);
+            setAlertContent({_id: uuid(), isShow:true, type:'SUCCESS', content:"video added to liked video list"})
+          }
+        } catch (err) {
+          setAlertContent({_id: uuid(), isShow:true, type:'ERROR', content:"Unexpected error please try again later !!"})
```

Comment:
fix: Here you can show diff alerts on the basis of status code that you get

Reply:
unable to manage due to time constraint